packages/twenty-front/src/pages/settings/data-model/constants/MetadataLabelalphanumericPattern.ts

```diff
@@ -0,0 +1 @@
+export const METADATA_LABEL_ALPHANUMERIC_PATTERN = /^[A-Za-z0-9\s]+$/;
```

Comment:
also this regex is too restrictive: $aaaa seems to be acceptable name for instance